cx-server-companion/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:11-alpine
+
+ARG npm_registry=https://registry.npmjs.org/
+ARG cx_server_base_uri=https://raw.githubusercontent.com/SAP/devops-docker-images/master/jenkins-master/REPLACE-ME
```

Comment:
Is the `REPLACE_ME` here and elsewhere in the PR intentionally or a left-over?

Reply:
Yes. This needs to be aligned. For example, the name of the docker repository is currently a place holder.